hack/validate-manifests/config.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+type Config struct {
+	SkipApplications map[string]bool
```

Comment:
Why are the `Skip*` config attributes maps? Wouldn't it be easier to have these as slices of strings? What purpose would be to provide `"myapp": false` ?

Reply:
It's much more convenient to look up a string in a map, than in a slice. If it's more clear we could make it a slice, but I would still make a `map[string]anything` out of it on start for faster and more convenient lookup.

Reply:
Gotcha, that makes sense.